### ISSUE TITLE
Multiple Homeys

### ISSUE THREAD

**Issue by jahnkenneth** [swttt/com.swttt.homekit#151]
Hi,
I have two Homeys and want to use Homeykit on both... 
Since the HomeKit code will be the same on both, this doesn’t work. 
Any advice?

**Comment by robertklep** [comment]
This won't work with the current app, it needs to generate a unique id for each Homey to make this work. I'll try and see if I can whip up a beta somewhere this week.

**Comment by jahnkenneth** [comment]
That would be splendid! :-D

**Comment by robertklep** [comment]
@jahnkenneth do you know how to run apps from the command line? If so, you can try running the current `beta` branch, which has a configurable "bridge identifier" setting (that needs to be unique for each Homey):
```
$ git clone --single-branch --branch beta https://github.com/swttt/com.swttt.homekit
$ cd com.swttt.homekit
$ athom app run
```

**Comment by jahnkenneth** [comment]
Installed, but still not able to add the second Homey. 
It does not show up in the pairing dialogue in the Home app. :-(

**Comment by robertklep** [comment]
Did you change the bridge identifier on your second Homey (in the Homeykit app settings), and restart the app after you did?

**Comment by jahnkenneth** [comment]
I did...
Does it maybe need a unique pairing code?

**Comment by robertklep** [comment]
Hmm, I realize now that the app may crash before you are able to change the identifier. I'll have to investigate that.

**Comment by robertklep** [comment]
No, it doesn't need a unique pairing code. Do you use a Homekit hub in your network?

**Comment by jahnkenneth** [comment]
Yes, i have multiple home pods av apple TV´s. One is acting as a hub.
Dosent seem like the app has crashed... Should i send you a log from the Homey app?

**Comment by robertklep** [comment]
When you run `athom app run`, it should log the identifier that it will use. Make sure that it's the identifier that you entered.

If that's okay, and the app logs `[HomekitApp] Started bridge` without any errors, it might be a discovery issue with Homekit, but that's sadly not something that I can fix (the Homekit protocol is a bit hit-or-miss in my experience, for some people discovery works great, for other people it just doesn't work at all).

**Comment by jahnkenneth** [comment]
The log:
─────────────── Logging stdout & stderr ───────────────
Warning: Trying to access `Homey.env.RESET_SETTINGS` but the environment variable has not been set!
2019-04-24 14:26:44 [log] [HomekitApp] starting server
2019-04-24 14:26:44 [log] [HomekitApp] Using "Soverom" as bridge identifier
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'Homey 3. etasje' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'Soverom' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'TV i stuen' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'godnatt til gammelhomey' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'godmorgen' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Found Sensor: Utendørs
created listener for - measure_temperature
created listener for - measure_humidity
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'One — Hovedbad' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'Inngangsdør' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'Infrared device (send only)' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'Infrared device (send only)' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'RM Mini (DC:76:00:77:0F:78)' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding '1. etasje' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding '2. etasje' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding '3. etasje' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'JK' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'VJ' (shouldn't be paired)
2019-04-24 14:26:44 [log] [HomekitApp] Not adding 'Sveabakken — Hovedsoverom' (shouldn't be paired)

Still doesn't show up the in the Home app... :-(

**Comment by robertklep** [comment]
Is that the entire log? I'm missing the "Started bridge" part.

**Comment by jahnkenneth** [comment]
Sorry...
Here is the rest. :-)

2019-04-24 14:28:36 [log] [HomekitApp] Not adding 'I morgen' (shouldn't be paired)
2019-04-24 14:28:36 [log] [HomekitApp] Started bridge
State Change - Utendørs - measure_humidity - 25
State Change - Utendørs - measure_temperature - 17.6
State Change - Utendørs - measure_humidity - 24

**Comment by robertklep** [comment]
Okay, so the bridge starts successfully. Which means it is a discovery issue. Are both of your Homey's connected to the same wireless network? I know that especially wifi meshing solutions can cause problems with Homekit discovery.

**Comment by jahnkenneth** [comment]
Yes, both are on the same network. 
I have a wifi mesh system in my house, so maybe that’s the problem. 
I have, however, never had any problems discovering the other hubs and accessories in my house. 
I just tried the other HomeKit app available in the app store (Sprut has updated his to support Homey 2.0), and that worked immediately...
But i would prefer to just use your app of course. :-)

**Comment by robertklep** [comment]
Hmm strange, no idea why it isn't being discovered. I don't have two Homey's so can't test that specific setup, but changing the bridge identifier works.
